crates/sui-indexer/src/apis/extended_api.rs

```diff
@@ -80,7 +80,7 @@ impl<S: IndexerStore> ExtendedApi<S> {
 
 #[async_trait]
 impl<S: IndexerStore + Sync + Send + 'static> ExtendedApiServer for ExtendedApi<S> {
-    async fn get_epoch(
+    async fn get_epochs(
```

Comment:
is there ts change?

Reply:
yes....I am expecting a few ts failure, will fix them now